src/main/java/org/folio/circulation/domain/validation/NotInItemStatusValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.domain.ItemStatus.CLAIMED_RETURNED;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import org.folio.circulation.domain.ItemStatus;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.support.Result;
+
+public final class NotInItemStatusValidator {
+
+  private NotInItemStatusValidator(){ }
+
+  public static Result<Loan> refuseWhenItemIsNotClaimedReturned(
+    Result<Loan> loanAndRelatedRecords) {
+
+    return loanAndRelatedRecords
+      .next(p -> refuseWhenItemIsNotInStatus(loanAndRelatedRecords, CLAIMED_RETURNED));
+  }
+
+  private static Result<Loan> refuseWhenItemIsNotInStatus(
+    Result<Loan> loanResult, ItemStatus status) {
+
+    return loanResult.failWhen(
+      records -> succeeded(!loanResult.value().getItem().isInStatus(status)),
```

Comment:
Personally, I might be tempted to introduce a `isNotInStatus` method to the item to avoid the negation here.

```suggestion
      records -> succeeded(!loanResult.value().getItem().isInStatus(status)),
```

(I'm also wondering if `in` and `notIn` would be more readable names for these methods)